webapp-django/crashstats/crashstats/tests/test_views.py

```diff
@@ -453,6 +456,16 @@ def mocked_processed_crash_get(**params):
         # shows up.
         assert "The crashing address." in content
 
+        metrics_mock.assert_timing(
+            "webapp.view.pageview",
+            tags=[
+                "status:200",
+                "ajax:false",
+                "api:false",
+                "path:report/index/_crashid_crash_id_",
```

Comment:
Oh, hrm... This might not work because it's not the full route but rather the part of the route defined in the matching resolver thing.

I need to add tests to the other views and see if it does what I want it to do. If it doesn't, I'll have to think of something else.